src/bootstrap/test.rs

```diff
@@ -2219,6 +2210,11 @@ fn prepare_cargo_test(
 ) -> Command {
     let mut cargo = cargo.into();
 
+    // Propegate `--bless` if it has not already been set/unset
+    if builder.config.cmd.bless() && !cargo.get_envs().any(|v| v.0 == "RUST_BLESS") {
+        cargo.env("RUST_BLESS", "Gesundheit");
```

Comment:
I like that `Gesundheit` bubbled up :smile:

Reply:
Aber selbstverständlich 🙂